### PR TITLE
feat: add live testnet testing instructions to readme

### PR DESCRIPTION
**Pull request by deanamiel**
Added instructions for running tests against live testnets to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +25,44 @@ npm run build
...
 
+## Live testnet testing
+
+1. Ensure that the contract deployments repository supports the testnet you will be using. Supported chains can be found [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/info/testnet.json).
+2. In the root folder of this repository create a new file named `keys.json`
+3. Within `keys.json` provide your private key for the account you will be using for testing. For some tests, such as the Axelar gateway tests, you may need to provide more than one private key. Next, you may optionally provide an API key for the testnet you will be using.
+
+    At this point the `keys.json` file should resemble the example file below:
+
+    ```json
```
Comment (milapsheth) [inline README.md:36]: remove spaces in the beginning

File: README.md [hunk 2]
```diff
@@ -24,6 +25,44 @@ npm run build
...
+            },
+            "CHAIN_NAME2": {
+                "api": "EXAMPLE_API_KEY"
+            }
+        },
+        "accounts": ["PRIVATE_KEY1", "PRIVATE_KEY2", "PRIVATE_KEY3"]
+    }
+    ```
+
+4. Ensure that your accounts corresponding to the private keys provided have sufficient native value on the testnet you will be using.
```
Comment (milapsheth) [inline README.md:50]: formatting is off which is why this shows up as red

File: README.md [hunk 3]
```diff
@@ -24,6 +25,44 @@ npm run build
...
+2. In the root folder of this repository create a new file named `keys.json`
+3. Within `keys.json` provide your private key for the account you will be using for testing. For some tests, such as the Axelar gateway tests, you may need to provide more than one private key. Next, you may optionally provide an API key for the testnet you will be using.
+
+    At this point the `keys.json` file should resemble the example file below:
+
+    ```json
+    {
+        "chains": {
+            "CHAIN_NAME1": {
+                "api": "EXAMPLE_API_KEY"
```
Comment (milapsheth) [inline README.md:40]: api key isn't needed

**Review by re1ro — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (deanamiel) [reply README.md:40]: removed

File: README.md [hunk 4]
```diff
@@ -24,6 +25,36 @@ npm run build
 npm run test
 ```
 
+## Live testnet testing
+
+1. Ensure that the contract deployments repository supports the testnet you will be using. Supported chains can be found [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/info/testnet.json).
```
Comment (milapsheth) [inline README.md:30]: you also need to update hardhat config in this repo to point to your local modified info file

Comment (deanamiel) [reply README.md:30]: I'll add instructions for this

File: README.md [hunk 5]
```diff
@@ -24,6 +25,63 @@ npm run build
...
+```
+
+4. In the root directory of this repository navigate to the `hardhat.config.js` file and modify the chains import line as shown below:
+
+```javascript
+const chains = require(`@axelar-network/axelar-contract-deployments/info/${env}.json`);
+const chains = require(`[LOCAL_PATH]/axelar-contract-deployments/info/${env}.json`);
+```
+
+5. In the root folder of this repository create a new file named `keys.json`
```
Comment (npty) [inline README.md:58]: does the `keys.json` need to put inside the `info` folder? (I noticed that from the require path in hardhat.config.js)

File: README.md [hunk 6]
```diff
@@ -24,6 +25,63 @@ npm run build
...
+6. Within `keys.json` provide your private key for the account you will be using for testing. For some tests, such as the Axelar gateway tests, you may need to provide more than one private key. Next, you may optionally provide an API key for the testnet you will be using.
+
+At this point the `keys.json` file should resemble the example file below:
+
+```json
+{
+    "accounts": ["PRIVATE_KEY1", "PRIVATE_KEY2", "PRIVATE_KEY3"]
+}
+```
```
Comment (npty) [inline README.md:67]: do we need to add [chains](https://github.com/axelarnetwork/axelar-contract-deployments/blob/b6d46f0a92d3bdd14652183ca10955ab89b70267/evm/.example.keys.json#L2) as well?

File: README.md (hunk 5 above)
Comment (milapsheth) [reply README.md:58]: switching to outside info folder in the other PR for consistency

File: README.md (hunk 6 above)
Comment (milapsheth) [reply README.md:67]: only needed for contract explorer verification

File: README.md (hunk 5 above)
Comment (deanamiel) [reply README.md:58]: good catch!

**Pull request merged**